thrust/system/cuda/detail/malloc_and_free.h

```diff
@@ -23,13 +23,16 @@
 #include <thrust/detail/raw_reference_cast.h>
 #include <thrust/detail/seq.h>
 #include <thrust/system/cuda/config.h>
-#ifdef THRUST_CACHING_DEVICE_MALLOC
-#include <cub/util_allocator.cuh>
-#endif
 #include <thrust/system/cuda/detail/util.h>
 #include <thrust/system/detail/bad_alloc.h>
 #include <thrust/detail/malloc_and_free.h>
 
+#ifdef THRUST_CACHING_DEVICE_MALLOC
```

Comment:
I don't see any documentation on this macro, is it still in use?

Reply:
No idea.